README.md

```diff
@@ -23,3 +23,19 @@ contextStorage: {
  - `projectID` - is the UUID of the project
  - `baseURL` - the root URL for the FlowForge Storage API
  - `token` - authentication token
+
+### Known Limitations
+- Currently, only async versions of context get and context set are supported
+  e.g. 
+  ```js
+  // SUPPORTED...
+  global.set("var1", "persistent", "i am the value", (err) => {
+     //do something
+  })
+  global.get("var1", "persistent", (err, val) => {
+    //do something with val
+  })
+  
+  // UNSUPPORTED...
+  // global.set("var1", "persistent", "i am the value")
+  // const value = global.get("var1", "persistent")
```

Comment:
There is no closing 3 back ticks (and I can't add it as a suggestion)

Reply:
resolved in [5fe1b9e](https://github.com/flowforge/flowforge-nr-persistent-context/pull/12/commits/5fe1b9e13198d9fb410fd272a8add0e1edea6ac6)